Source/DafnyCore/Backends/GoLang/GoCodeGenerator.cs

```diff
@@ -10,15 +10,27 @@
 using System.Numerics;
 using System.IO;
 using System.Diagnostics.Contracts;
-using System.Collections.ObjectModel;
 using System.Text.RegularExpressions;
+using DafnyCore;
 using JetBrains.Annotations;
+using Tomlyn.Model;
 using static Microsoft.Dafny.ConcreteSyntaxTreeUtils;
 
 namespace Microsoft.Dafny.Compilers {
   class GoCodeGenerator : SinglePassCodeGenerator {
+    //TODO: This is tentative, update this to point to public module once available.
```

Comment:
Note to self: come back to this after reviewing tests

Reply:
The current `DafnyRuntimeGo` is not compatible with the Go Modules structure. I also can't simply change the existing one because we are also supporting GOPATH for the time being. The ideal option (and required one soon enough) is to publish the `DafnyRuntimeGo` as a go module.

For now, I've moved out my implementation of `DafnyRuntimeGo` as a Go modules to the [Source/DafnyRuntime/DafnyRuntimeGo-gomod](https://github.com/dafny-lang/dafny/tree/master/Source/DafnyRuntime/DafnyRuntimeGo-gomod) directory, based on our discussion.